source/channel_bootstrap.c

```diff
@@ -410,8 +438,7 @@ static void s_on_client_channel_on_setup_completed(struct aws_channel *channel,
         err_code);
 
 error:
```

Comment:
trivial: maybe move the LOGF_ERROR statement below the `error:` label so it prints every time